pkg/util/extender/unstructured.go

```diff
@@ -183,8 +183,7 @@ func ObjectEqual(newUnstruct, oldUnstruct *unstructured.Unstructured) bool {
 	annotations := map[string]string{}
 	for k, v := range oldUnstruct.GetAnnotations() {
 		if k == annotation.AnnLastAppliedConfigKey ||
-			k == annotation.AnnRestartTimestamp ||
-			k == annotation.AnnLastReplicas {
+			k == annotation.AnnRestartTimestamp {
```

Comment:
why remove this annotation

Reply:
@MegaByte875 I removed this check to fix the Graphd is stuck in "Scale In" though the nebula cluster is ready issue. This is due to new_replicas not equal to last_replicas which is because last_replicas wasn't updated to the correct value. So we shouldn't ignore the annotation AnnLastReplicas when checking equality so we can perform an update.

Reply:
According to @MegaByte875 `k == annotation.AnnLastReplicas` was added to fix a bug. Will leave alone for know. @MegaByte875 will look into why the replicas are not equal.